### PR TITLE
Add Disney and Disney Plus as easy

### PR DESCRIPTION
**Pull request by WebSnke**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -3856,12 +3856,30 @@
...
+        "domains": [
+            "https://www.disney.co.uk",
+            "https://www.disney.com",
+            "https://www.disney.de",
+            "https://www.disney.es",
+            "https://www.disney.fi/",
+            "https://www.disney.fr",
+            "https://www.disney.my/",
+            "https://www.disney.pt/",
+            "https://support.disney.com"
```
Comment (tupaschoal) [inline _data/sites.json:3872]: Actually these domains should not have the "https://", it was just an oversight from the entry below.
```suggestion
            "disney.co.uk",
            "disney.com",
            "disney.de",
            "disney.es",
            "disney.fi",
            "disney.fr",
            "disney.my",
            "disney.pt",
            "support.disney.com"
```

File: _data/sites.json [hunk 2]
```diff
@@ -3856,12 +3856,30 @@
...
+    
     {
         "name": "Disney+",
         "url": "https://help.disneyplus.com/csp?id=csp_article_content&sys_kb_id=bf6bf352db101558331d6fddd39619c7",
         "difficulty": "medium",
+        "note": "Your Disney+ account is different from your Disney account. Deleting your Disney+ account will not delete your Disney account.",
         "domains": [
-            "https://www.disneyplus.com"
+            "https://www.disneyplus.com",
             "https://help.disneyplus.com"
```
Comment (tupaschoal) [inline _data/sites.json:3883]:
```suggestion
            "disneyplus.com",
            "help.disneyplus.com"
```

File: _data/sites.json [hunk 3]
```diff
@@ -3855,6 +3855,34 @@
...
+    },
+    
+    {
+        "name": "Disney+",
+        "url": "https://help.disneyplus.com/csp?id=csp_article_content&sys_kb_id=bf6bf352db101558331d6fddd39619c7",
+        "difficulty": "medium",
+        "note": "Your Disney+ account is different from your Disney account. Deleting your Disney+ account will not delete your Disney account.",
+        "domains": [
+            "https://www.disneyplus.com",
+            "https://help.disneyplus.com"
```
Comment (tupaschoal) [inline _data/sites.json:3883]:
```suggestion
            "disneyplus.com",
            "help.disneyplus.com"
```

File: _data/sites.json [hunk 4]
```diff
@@ -3855,6 +3855,34 @@
...
+            "disney.pt",
+            "support.disney.com"
+        ]
+    },
+    
+    {
+        "name": "Disney+",
+        "url": "https://help.disneyplus.com/csp?id=csp_article_content&sys_kb_id=bf6bf352db101558331d6fddd39619c7",
+        "difficulty": "medium",
+        "note": "Your Disney+ account is different from your Disney account. Deleting your Disney+ account will not delete your Disney account.",
```
Comment (tupaschoal) [inline _data/sites.json:3880]:
```suggestion
        "notes": "Your Disney+ account is different from your Disney account. Deleting your Disney+ account will not delete your Disney account.",
```

File: _data/sites.json [hunk 5]
```diff
@@ -3855,6 +3855,34 @@
             "users.rust-lang.org"
         ]
     },
+    
+    {
+        "name": "Disney",
```
Comment (tupaschoal) [inline _data/sites.json:3860]:
```suggestion

    {
        "name": "Disney",
```

File: _data/sites.json [hunk 6]
```diff
@@ -3855,6 +3855,34 @@
...
+            "disney.es",
+            "disney.fi",
+            "disney.fr",
+            "disney.my",
+            "disney.pt",
+            "support.disney.com"
+        ]
+    },
+    
+    {
```
Comment (tupaschoal) [inline _data/sites.json:3876]:
```suggestion
    },

    {
```

**Review by tupaschoal — COMMENTED** [review]
Some trailing spaces to remove

**Review by tupaschoal — APPROVED** [review]
I was surprised we didn't have Disney yet.

**Comment by WebSnke** [comment]
> I was surprised we didn't have Disney yet.

Yeah, me too.

**Pull request merged**